trade-finance-manager-ui/server/api.js

```diff
@@ -457,6 +457,24 @@ const getLatestCompletedAmendmentByDealId = async (dealId) => {
   }
 };
 
+const getParty = async (partyUrn) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${tfmAPIurl}/v1/party/urn/${partyUrn}`,
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
do we need this? conscious we have it everywhere but the default should be application/json